### PR TITLE
Fixed payment isn't initiated on lndrest

### PR DESCRIPTION
**Pull request by skrafft**
Closes #2062

### CODE REVIEW

File: lnbits/wallets/lndrest.py [hunk 1]
```diff
@@ -195,6 +195,8 @@ async def get_payment_status(self, checking_id: str) -> PaymentStatus:
                             if "message" in line["error"]
                             else line["error"]
                         )
+                        if line["error"]["code"] == 5 and line["error"]["message"] == "payment isn't initiated":
```
Comment (prusnak) [inline lnbits/wallets/lndrest.py:198]: I think the following is more correct:

```suggestion
                        if line["error"].get("code") == 5 and line["error"].get("message") == "payment isn't initiated":
```

**Review by prusnak — CHANGES_REQUESTED** [review]

**Comment by skrafft** [comment]
@prusnak Thanks, I changed the PR

**Comment by callebtc** [comment]
Nice one. Could you please run the formatter? `make format` and push again please.

**Comment by skrafft** [comment]
Done, sorry for the mistake

**Review by dni — APPROVED** [review]

**Comment by prusnak** [comment]
@callebtc I don't understand the edit in the original post - does that mean that this PR supersedes PR https://github.com/lnbits/lnbits/pull/2062?

**Review by prusnak — APPROVED** [review]

**Comment by prusnak** [comment]
> #2062

I think this should close https://github.com/lnbits/lnbits/issues/1913 not https://github.com/lnbits/lnbits/pull/2062

**Pull request merged**